.travis.yml

```diff
@@ -48,7 +48,8 @@ before_install:
   - sudo chown -R elasticsearch:elasticsearch /etc/default/elasticsearch
   - sudo systemctl start elasticsearch
 install:
-  - bundle install --path vendor/bundle
+  - bundle config set path 'vendor/bundle'
```

Comment:
This was moved bc of a deprecation warning that stated this is the new expected way to set the path.